src/include/daos/btree.h

```diff
@@ -443,6 +443,44 @@ struct btr_instance {
 	btr_ops_t			*ti_ops;
 };
 
+/**
+ * Inline data structure for embedding the key bundle and key into an anchor
+ * for serialization.
+ */
+#define	EMBEDDED_KEY_MAX	100
+struct btr_embedded_key {
+	/** Inlined iov key references */
+	uint32_t	ek_size;
+	/** Inlined buffer the key references*/
+	unsigned char	ek_key[EMBEDDED_KEY_MAX];
+};
+D_CASSERT(sizeof(struct btr_embedded_key) == DAOS_ANCHOR_BUF_MAX);
```

Comment:
```suggestion
};

```

Reply:
probably need a space here to avoid this?